modules/default/weather/providers/openweathermap.js

```diff
@@ -1,13 +1,5 @@
 /* global WeatherProvider, WeatherObject */
 
-/* MagicMirror²
- * Module: Weather
- *
- * By Michael Teeuw https://michaelteeuw.nl
- * MIT Licensed.
- *
- * This class is the blueprint for a weather provider.
```

Comment:
Helpful line should stay in

Reply:
helpful? This is only a copy from `weatherprovider.js`

Reply:
added useful comment with link instead

Reply:
ups, my mistake :-)